modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/utils/StreamHandler.java

```diff
@@ -0,0 +1,57 @@
+package org.wso2.mdm.agent.proxy.utils;
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class StreamHandler {
```

Comment:
Having a name similar to "StreamHandlerUtil" instead of "StreamHandler" would have been cooler.